### PR TITLE
feat: created quiz page

### PR DESCRIPTION
**Pull request by Shubhamkr2610**
Quiz page 
files to check : 
src/pages/Quiz.jsx
src/pages/quiz.css
src/context/quizContext.js

image

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -12,8 +12,8 @@ function App() {
         <Header />
         <Routes>
           <Route path="/" exact element={ <Home /> }/> 
-          <Route path="/Quiz" exact element={ <Quiz /> }/>
-          <Route path="/Result" exact element={ <Result /> }/>
+          <Route path="/quiz" exact element={ <Quiz /> }/>
+          <Route path="/result" exact element={ <Result /> }/>
```
Comment (arjun-computer-geek) [inline src/App.js:16]: Add 404 not found page too

File: src/components/quizCard/QuizCard.css [hunk 2]
```diff
@@ -28,8 +28,11 @@
 .start-quiz{
     font-family: "Poppins", sans-serif;
     border: none;
-    height: 2rem;
+    display: block;
+    text-align: center;
+    height: 2.5rem;
     border-radius: 5px;
+    /* background:  yellow; */
```
Comment (arjun-computer-geek) [inline src/components/quizCard/QuizCard.css:35]: Remove commented code

File: src/pages/quiz-page/Quiz.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
+
+.quiz-question-container{  
+    height: 60vh;
+    width: 50%;
+    display: flex;
+    margin: auto;
```
Comment (arjun-computer-geek) [inline src/pages/quiz-page/Quiz.css:6]: Is there need to use margin: auto here? Because you are using display : flex here?

**Review by arjun-computer-geek — APPROVED** [review]
Code looks good and ui too. Nice 👍

**Review by sanjitsarkar — APPROVED** [review]
Code looks clean and UI looks great

File: src/pages/quiz-page/Quiz.css [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+.next-button{
+    padding: 5px;
+    width: 80%;
+    border: 2px solid var(--quiz-color-primary);
+    color: var(--quiz-color-primary);
+    border-radius: 4px;
+    font-family: "Poppins", sans-serif;
+}
+
+.next-button:hover{
```
Comment (Rockyraven) [inline src/pages/quiz-page/Quiz.css:43]: there is no meaning of putting font family multiple times 
you can put in body

File: src/pages/quiz-page/Quiz.jsx [hunk 5]
```diff
@@ -1,7 +1,76 @@
...
 export const Quiz = () => {
+  const { questionCollection } = useQuiz()
+  const [questionindex, setQuestionIndex] = useState(0);
+  const [question, setQuestion] = useState("");
+  const [option, setOption] = useState([]);
+  const [selectedQuestion, setSelectedQuestion] = useState(null);
+  const [score, setScore] = useState(0);
+
+
+  const nextQuestion = () => {
```
Comment (Rockyraven) [inline src/pages/quiz-page/Quiz.jsx:13]: you can use here reducer 
it will reduce your line of code

**Review by Rockyraven — APPROVED** [review]
do some minar changes
other wise your file handling system is systematic 
and code is clean and easy to readable

**Pull request merged**